buildSrc/src/main/kotlin/Project.kt

```diff
@@ -1,7 +1,7 @@
 object AndroidVersions {
   const val minSdkVersion = 19
-  const val targetSdkVersion = 28
-  const val compileSdkVersion = 28
+  const val targetSdkVersion = 30
```

Comment:
Doesn't it cause a crash for Android 30 devices because of okhttp? https://github.com/mapbox/mapbox-maps-android/issues/55

Reply:
we are now using 4.x :tada: